src/approx_const.rs

```diff
@@ -13,14 +12,25 @@ declare_lint! {
      is found; suggests to use the constant"
 }
 
-const KNOWN_CONSTS : &'static [(f64, &'static str)] = &[(f64::E, "E"), (f64::FRAC_1_PI, "FRAC_1_PI"),
-    (f64::FRAC_1_SQRT_2, "FRAC_1_SQRT_2"), (f64::FRAC_2_PI, "FRAC_2_PI"),
-    (f64::FRAC_2_SQRT_PI, "FRAC_2_SQRT_PI"), (f64::FRAC_PI_2, "FRAC_PI_2"), (f64::FRAC_PI_3, "FRAC_PI_3"),
-    (f64::FRAC_PI_4, "FRAC_PI_4"), (f64::FRAC_PI_6, "FRAC_PI_6"), (f64::FRAC_PI_8, "FRAC_PI_8"),
-    (f64::LN_10, "LN_10"), (f64::LN_2, "LN_2"), (f64::LOG10_E, "LOG10_E"), (f64::LOG2_E, "LOG2_E"),
-    (f64::PI, "PI"), (f64::SQRT_2, "SQRT_2")];
-
-const EPSILON_DIVISOR : f64 = 8192f64; //TODO: test to find a good value
+// Tuples are of the form (name, lower_bound, upper_bound)
```

Comment:
I think the comment should also note that the bounds are exclusive, like `excl_lower_bound, excl_upper_bound` or something. I had initially planned for the bounds to be inclusive, because engineers apparently like to use 3.14 in lieu of PI, and I would be a bit puzzled if the lint triggered at 3.1391, but maybe that's just me.